legal-api/src/legal_api/services/namex.py

```diff
@@ -69,6 +69,11 @@ def query_nr_number(identifier: str):
             'Authorization': 'Bearer ' + token
         })
 
+        # Return if not successful so downstream can handle the error
+        if nr_response.status_code != 200:
+            return nr_response
+
+        # Return the json if successful
         return nr_response.json()
```

Comment:
I think converting this to json is problematic because of the other places calling `query_nr_number`.  I noticed you had updated `namerequest.py` but I do see a bunch of other places where `query_nr_number` is called.  Potentially this may be an issue I'm thinking.

Reply:
Ah ok. I see two instances where it might call `.json()` before validations. 
So your suggesting i remove those two instances since we are already doing the conversion?

Reply:
As per our conversation in RC, I think having `query_nr_number` return an object as opposed to json would be more ideal.  The consumer of `query_nr_number` can decide how in what format it wants to return the object if that is required.

Reply:
Sounds good, refactoring! 👍